book/forms.rst

```diff
@@ -77,8 +77,8 @@ from inside a controller::
     // src/AppBundle/Controller/DefaultController.php
     namespace AppBundle\Controller;
 
-    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
     use AppBundle\Entity\Task;
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
```

Comment:
Not sure what is the convention, but I personally try to sort alphabetically (groups included if needed) when possible.